.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,43 @@
+name: omise-go code-coverage
+
+on:
+  push:
+
+jobs:
+  sonarcloud:
+    name: SonarCloud
+    runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      WORKDIR: src/github.com/${{ github.repository }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go ${{ matrix.go }}
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.15" # Old Go version needed to run "go test"
```

Comment:
@aashishgurung We might need to support `go mod` to test on a more recent version of Go.